commands/duck.js

```diff
@@ -0,0 +1,28 @@
+const got = require('got');
+
+module.exports = {
+	name: 'dog',
```

Comment:
duck

Reply:
Fixed in 4ecaa183ede7d7ffc730ff4b5c1b08666f0d613d